operations/exports.md

```diff
@@ -0,0 +1,158 @@
+# Exports
+
+> ### Restricted!
+> These operations are currently in beta-test and as such are subject to change.
+
+## Get all exports
+
+Get exports by provided `ExportIds`. Note this operation supports [Portfolio Access Tokens](../guidelines/multi-property.md).
+
+### Request
+
+`[PlatformAddress]/api/connector/v1/exports/getAll`
+
+```json
```

Comment:
Remove json. We don't do that, we're not sure if there wasn't any issues with it 🤔 that must by checked by mike. Do it everywhere.

Reply:
Yeah, I saw there's `javascript` instead. Probably should check whether GitBook can crunch it.